docs/serpen_aqua_ubi.md

```
@@ -0,0 +1,256 @@
+     run: |
+       # Compute SHA256 and store in a file (platform-specific command)
+       if [[ "$RUNNER_OS" == "Windows" ]]; then
+         certutil -hashfile serpen_${{ matrix.target_os }}_${{ matrix.target_arch }}.zip SHA256 > checksum.txt
+         findstr /v "hash" checksum.txt > serpen_${{ matrix.target_os }}_${{ matrix.target_arch }}.sha256
+       else
+         sha256sum serpen_${{ matrix.target_os }}_${{ matrix.target_arch }}.tar.gz > serpen_${{ matrix.target_os }}_${{ matrix.target_arch }}.sha256
+       fi
+   ```
+
+   **Explanation:** On Linux/macOS runners, the step uses `sha256sum` to generate a checksum and writes it to a file named `serpen_<os>_<arch>.sha256`. On Windows, we use the built-in `certutil` to get the SHA256 (redirecting output and filtering out extraneous text) and save it similarly. Each `.sha256` file will contain the hash and the filename. For example, `serpen_linux_amd64.sha256` might contain:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)